marketplace-service/src/main/java/com/axonivy/market/controller/ProductController.java

```diff
@@ -113,6 +113,30 @@ public ResponseEntity<Message> syncProducts(@RequestHeader(value = AUTHORIZATION
     return new ResponseEntity<>(message, HttpStatus.OK);
   }
 
+  @PutMapping(SYNC_ONE_PRODUCT)
+  @Operation(hidden = true)
+  public ResponseEntity<Message> syncOneProduct(
+      @RequestHeader(value = AUTHORIZATION) String authorizationHeader,
+      @RequestParam(value = ID) @Parameter(
+          description = "Product Id is defined in meta.json file", example = "a-trust") String productId,
+      @RequestParam(value = MARKET_ITEM_PATH) @Parameter(
+          description = "Item folder path of the market in https://github.com/axonivy-market/market",
+          example = "market/connector/a-trust") String marketItemPath) {
```

Comment:
Do we need to input the marketItemPath here ? because I checked that product also have field "marketDirectory".

Can we re-use this field from product ?